src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -704,14 +705,14 @@ private EntityHandle GetTypeReferenceOrSpecificationHandle(Type type)
             return typeHandle;
         }
 
-        private EntityHandle GetResolutionScopeHandle(Type type)
+        private (EntityHandle, string?) GetResolutionScopeAndNamespace(Type type)
```

Comment:
I would inline this logic to the one place where it is called from. I think it would be simpler and easier to read that way.

Reply:
Done, thanks!